tests/nat/mcp/test_mcp_session_management.py

```
@@ -500,6 +548,301 @@ async def test_manual_cleanup_returns_correct_count(self, function_group):
+        mock_client = AsyncMock()
+
+        # Make __aenter__ hang indefinitely
+        async def hanging_aenter(self):
+            await asyncio.sleep(1000)  # Never completes
+            return mock_client
+
+        mock_client.__aenter__ = hanging_aenter
+        mock_client.__aexit__ = AsyncMock(return_value=None)
+
+        with patch('nat.plugins.mcp.client_base.MCPStreamableHTTPClient', return_value=mock_client):
+            with pytest.raises(RuntimeError, match="Session client initialization timed out after 60.0s"):
```

Ruff (0.13.3)

Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)